docs/sphinx_preprocessor.py

```diff
@@ -0,0 +1,34 @@
+import json
```

Comment:
Nit: how about adding a shebang? We had it in the previous script.

Reply:
This script is not supposed to be run manually, so I don't see how shebang helps?

Reply:
Makes sense not to do it.